pkg/prometheus/operator.go

```diff
@@ -1511,6 +1730,25 @@ func (c *Operator) selectServiceMonitors(p *monitoringv1.Prometheus) (map[string
 		})
 	}
 
+	// If denied by Prometheus spec, filter out all service monitors that access
+	// the file system.
```

Comment:
this will also filter out service monitors which have an invalid tls configuration. can you add that to the comment here?